pkg/config/config_local_test.go

```diff
@@ -18,7 +18,7 @@ import (
 var _ = Describe("Config Local", func() {
 	BeforeEach(beforeEach)
 
-	It("should fail on invalid NetworkConfigDir", func() {
```

Comment:
A little torn on whether to remove the existing tests or to negate them. I'm in favor of negating them howsoever weird they may seem since we might still be testing _some_ functionality. Wdyt?